rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json

```diff
@@ -509,11 +594,7 @@
         "hint": "Date of onset of symptoms",
         "expanded": false,
         "min_date": "today-100y",
-        "max_date": "today",
-        "v_required": {
-          "value": "true",
-          "err": "This field is required"
-        }
+        "max_date": "today"
```

Comment:
Could we add the sampler ID and phone number https://github.com/onaio/rdt-standard/issues/799#issuecomment-784321116?

Reply:
I have added a sampler id and phone number on sample-collection-form https://github.com/onaio/rdt-standard/blob/440b5fb4e1e0cc4ba215a600e727728f32dc4051/rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json#L73-L99

Reply:
cool